_episodes/05-access-data.md

```diff
@@ -247,12 +247,12 @@ print(item.properties)
 > > {: .language-python}
 > >
 > > ~~~
-> > 4
+> > 2
```

Comment:
ah thank so much! I'll use that.

Reply:
I checked the o:cloud cover values and it looks like two are above 10 and 2 are below, so looks like this is working. Not sure what v0 returned in the past, b ut unfortunately can't get a valid search to complete with the old endpoint.

Reply:
Ah, weird behaviour for the v0 endpoint. From some record files I see that the cloud cover values returned were slightly different (8.65, 0, 0.03, and 5), they might have been estimated from different Sentinel-2 metadata. Here, the easier thing might be to modify the threshold in the exercise and set it to 15%, so that the search leads to the the same item collection (and thus, we have less elements to adapt in the rest of the lesson..).

Reply:
done!